src/Umbraco.Web.BackOffice/Controllers/MemberController.cs

```diff
@@ -453,13 +459,56 @@ private async Task<ActionResult<bool>> UpdateMemberAsync(MemberSave contentItem)
                     return new ValidationErrorResult(validatePassword.Errors.ToErrorMessage());
                 }
 
-                string newPassword = _memberManager.HashPassword(contentItem.Password.NewPassword);
-                identityMember.PasswordHash = newPassword;
+                Attempt<int> intId = identityMember.Id.TryConvertTo<int>();
+                if (intId.Success == false)
+                {
+                    return new ValidationErrorResult("Member ID was not valid");
+                }
+
+                IMember foundMember = _memberService.GetById(intId.Result);
+                if (foundMember == null)
+                {
+                    return new ValidationErrorResult("Member was not found");
+                }
+
+                IUser currentUser = _backOfficeSecurityAccessor.BackOfficeSecurity.CurrentUser;
+                // if the current user has access to reset/manually change the password
```

Comment:
@Shazwazza now the logic has been moved to the controller, I don't know if we need to check the section access for changing member passwords?